src/components/Calendar/doubleCalendar/yearSelect/index.js

```diff
@@ -49,24 +65,39 @@ export default function YearSelect(props) {
                 ref={selectRef}
                 value={currentYear}
                 editMode={isEditMode}
+                conClick={onClick}
```

Comment:
conClick?

Reply:
that was a mistake... i din't see that... LOL...